aidants_connect_web/views/mandat.py

```diff
@@ -240,6 +219,32 @@ def attestation_projet(request):
 @login_required
 @activity_required
 def attestation_final(request):
+    return __attestation_visulisation(request, settings.MANDAT_TEMPLATE_PATH)
+
+
+@login_required
+@activity_required
+def attestation_visualisation(request, mandat_id):
+    mandat = Mandat.objects.filter(pk=mandat_id)
+    if mandat.count() != 1:
+        return render(
+            request,
+            "aidants_connect_web/mandat_not_found.html",
+            {"message": f"Pas de mandat trouvé avec l'identifiant {mandat_id}"},
+        )
+
+    template = mandat.first().get_template_path(request.user)
+    if template is not None:
+        return __attestation_visulisation(request, template)
+
+    return render(
+        request,
+        "aidants_connect_web/mandat_not_found.html",
+        {"message": "Impossible de récupérer le document original pour ce mandat"},
+    )
+
+
+def __attestation_visulisation(request, template: str):
```

Comment:
Ici aussi on peut ajouter le petit `a` perdu ! (Sinon kaboum)
```suggestion
def __attestation_visualisation(request, template: str):
```